molmod/forms.py

```diff
@@ -91,6 +93,24 @@ def cover_check(form, field):
             raise ValidationError('Value between 0 and 100, please.')
 
 
+def file_check(form, field):
+    file = field.data
+    if file is not None:
+        filename = file.filename
+        try:
+            ext = os.path.splitext(filename)[1]
+            APP.logger.debug(f'ext is {ext}')
+        except Exception:
+            return None
+        regex = "([^\\s]+(\\.(?i)(xlsx|tar\\.gz|tar\\.bz2|tar\\.lz))$)"
```

Comment:
This is fine. I'd have just checked for ext in a list of approved extensions, but this is not wrong.

Reply:
Sure, a list sounds good. Feel free to edit.